nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -14,6 +14,9 @@ let
     ${cfg.extraConfig}
   '';
   qemuConfigFile = pkgs.writeText "qemu.conf" ''
+    ${optionalString cfg.qemuOvmf ''
+      nvram = ["${pkgs.OVMF}/FV/OVMF_CODE.fd:${pkgs.OVMF}/FV/OVMF_VARS.fd"]
```

Comment:
No, libvirtd copies the VARS (or the whole firmware) into /var/lib/libvirt/qemu/nvram when creating VMs. Here is the copy loop: https://github.com/libvirt/libvirt/blob/master/src/qemu/qemu_process.c#L3995

Also I tested this successfully.